debian/tests/haschangelog

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python3
+
+"""Test if associated PR has a changelog entry for the changes.
+
+"""
```

Comment:
Minor style nit: The closing triple quote can go right after the period (i.e. all on the same line).